consensus/src/round_manager_test.rs

```diff
@@ -104,7 +109,11 @@ impl NodeSetup {
                 signer.private_key().clone(),
                 waypoint,
             );
-            let safety_rules_manager = SafetyRulesManager::new_local(author, safety_storage);
+            let safety_rules_manager = SafetyRulesManager::new_local(
+                author,
+                safety_storage,
+                Ed25519PrivateKey::generate_for_testing().public_key(),
```

Comment:
probably repeating feedback from before, but this /must/ come from safety_storage not as an extra input parameter.